frontend/app/chat/[chatId]/hooks/useChat.ts

```diff
@@ -55,7 +55,12 @@ export const useChat = () => {
         //TODO: update chat list here
       }
 
-      void track("QUESTION_ASKED");
+      if (currentBrain?.id === undefined) {
```

Comment:
I think we should start using the properties: brain_id, prompt_id
If undefined that means we're headless
Wdyt ?